math/src/utils/mod.rs

```diff
@@ -199,6 +199,25 @@ pub fn log2(n: usize) -> u32 {
     n.trailing_zeros()
 }
 
+// CONVERSION FUNCTIONS
+// ================================================================================================
+
+/// Converts a slice of bytes into a field element. Pads the slice if it is smaller than the number
+/// of bytes needed to represent an element.
+///
+/// # Panics
+/// Panics if the length of `bytes` is smaller than the number of bytes needed to encode an element.
+pub fn bytes_to_element_with_padding<B: StarkField>(mut bytes: Vec<u8>) -> B {
```

Comment:
Good idea - named it `from_byte_vec_with_padding()` to make it clear that it tries to add padding before conversion. Otherwise I found it confusing why that method existed on top of the `TryFrom` conversion from slice to `StarkField`